### PR TITLE
Update documentation on testing with Cypress

### LINKED ISSUE #433: Update documentation to include testing in Cypress

### PR DESCRIPTION
**Pull request by crisner**
Fixes #433  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Fantastic!!!! 🎉

File: README.md [hunk 1]
```diff
@@ -95,11 +95,13 @@ A leaflet plugin that has a collection of layers containing environmental data p
...
 ### _Instructions for a developer_
 1. Install grunt - https://gruntjs.com/installing-grunt.
-2. Make the changes you are working on in respective /src files.
+2. Make the changes you are working on in the respective /src files.
 3. Run `grunt build` to generate files in the /dist directory.
 4. Run `grunt transpile` to transpile es6 code and copy files needed to run the tests to the /dist directory.
 5. Run `grunt jasmine` to run tests on the LEL layers and ensure they pass.
-6. Test your changes on a browser by opening `examples/index.html`.
+6. Run `npm run start` to start a local server.
+7. Run `npm run cy:run:chrome` to run e2e and integration tests.
```
Comment (sagarpreet-chadha) [inline README.md]: As a follow up, let' combine these 2 commands into one command:
`npm run cypress` which will be equal to `npm run start` && `npm run cy:run:chrome` in package.json. Thanks!
Can be a FTO a well!

Comment (crisner) [reply README.md]: Awesome! I'll create an FTO for this.

**Comment by crisner** [comment]
> Also we can run cypress tests in headless mode, we should add that to documentation as well, because not everyone will want to open separate server for running cypress tests 😄

Unfortunately, running Cypress tests in headless mode doesn't seem to work in these tests. Tried initially when I was setting up the tests to run in Travis. I am assuming it is because a child window is used. I had to incorporate the use of a child window as the default window did not like any change in the hash.

**Pull request merged**